.cruft.json

```diff
@@ -0,0 +1,20 @@
+{
+  "template": "https://github.com/PrefectHQ/prefect-collection-template",
+  "commit": "e11a3be195f24f60ed3f564dfccb40170ee7c3fa",
+  "checkout": null,
+  "context": {
+    "cookiecutter": {
+      "full_name": "Arthur Dent",
+      "email": "arthur.dent@example.com",
+      "github_organization": "arthur_dent",
+      "collection_name": "prefect-lakefs",
+      "collection_slug": "prefect_lakefs",
+      "collection_short_description": "Prefect Collection Template contains all the boilerplate that you need to create a Prefect collection.",
```

Comment:
change short desc to something meaniful.
keeps short desc consistent accross cruft updates.